File: service-backbase-java/src/main/java/org/example/backend/sampler/UserLoginSampler.java
package org.example.backend.sampler;

import org.example.backend.model.request.UserLoginReqBody;
import org.example.backend.model.request.UserLoginReqBody.User;

public class UserLoginSampler {

  public static UserLoginReqBody fullInput(String email, String password) {
    var user = new User()
        .email(email)
        .password(password);
    return new UserLoginReqBody().user(user);
  }

}

File: service-backbase-kotlin/src/main/kotlin/org/example/backend/sampler/UserLoginSampler.kt
package org.example.backend.sampler

import org.example.backend.model.request.User
import org.example.backend.model.request.UserLoginReqBody

object UserLoginSampler {

    fun fullInput(email: String?, password: String?): UserLoginReqBody? {
        return UserLoginReqBody(
                user = User(
                        email,
                        password
                )
        );
    }
}